config/200-clusterrole.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 The Sigstore Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cosigned-rbac
+rules:
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["create"]
+
+  # Allow the reconciliation of exactly our validating webhook.
+  - apiGroups: ["admissionregistration.k8s.io"]
```

Comment:
Why do we need all these permissions ? In the original chart, we don't need more than those https://github.com/sigstore/helm-charts/pull/10/files#diff-dba06e52f4da92f91dbb6c70da49d6d5c18f8822eb0516d76fba88c436689a0eR17.

Reply:
@mattmoor I was referring to all the yaml files in general. We decided to move the yaml manifests to the helm-charts repo.
cc @cpanato @dlorenc

Reply:
I'm happy to help rationalize things here, I just hadn't seen the "dots" connected for how y'all plan to do e2e testing with the configuration in a separate repo.  I really wanted e2e tests to help validate things here.

One thing I've done elsewhere is vendor the config files across repos, which is another possible option.